tools/autograd/derivatives.yaml

```diff
@@ -895,7 +895,10 @@
   self: soft_margin_loss_backward(grad, self, target, reduction)
 
 - name: relu(Tensor self)
-  self: threshold_backward(grad, self, 0, 0)
+  self: threshold_backward(grad, self, 0)
+
+- name: relu_(Tensor self)
```

Comment:
err wait do we actually need this?  I forget if we handle this automatically for native functions (with the above definition) or only if it's non-native?